install/docker/debian-docker-install.md

```diff
@@ -0,0 +1,121 @@
+# Install Docker for Kata Containers on Debian
+
+> **Note:**
+>
+> - This guide assumes you have
+>   [already installed the Kata Containers packages](../debian-installation-guide.md).
+> - this guide allows for installation with `systemd` or `sysVinit`init systems 
+> - notes are also provided for running multiple runtimes
+
+1. Install the latest version of Docker with the following commands:
+
+   > **Note:** This step is only required if Docker is not installed on the system.
+
+   ```bash
+   $ sudo apt-get install apt-transport-https ca-certificates curl gnupg2 software-properties-common  
```

Comment:
This needs to be `sudo apt-get -y install` to ensure it can be run non-interactively (by the CI).

Reply:
This command still needs a `-y` so we can run it non-interactively in our CI.